tests/test_widgets.py

```diff
@@ -54,6 +54,18 @@ def test_widget_plane():
     p.add_plane_widget(callback=func, pass_widget=True, implicit=False)
     p.close()
 
+    p = pyvista.Plotter()
+    func = lambda normal, origin: normal # Does nothing
+    p.add_mesh(mesh)
+    p.add_plane_widget(callback=func, assign_to_axis='z', implicit=True)
+    p.close()
```

Comment:
Do we need the `NO_PLOTTING` skips when there's no `p.show()`, only `p.close()`? I never know :)

Reply:
Yes, we still need it regardless. If the system doesn't support plotting (no virtual display or graphics lib) then instantiating the plotter will fail.

A render window is created regardless of the `show()` call (show simply displays what already exists)

Reply:
I see, thanks for the explanation :)